src/Microsoft.AspNetCore.Mvc.ApiExplorer/DefaultApiDescriptionProvider.cs

```diff
@@ -383,7 +409,9 @@ private string GetRelativePath(RouteTemplate parsedTemplate)
                 {
                     if (part.IsLiteral)
                     {
-                        currentSegment += part.Text;
+                        currentSegment += _routeOptions.LowercaseUrls ?
+                            part.Text.ToLowerInvariant() :
```

Comment:
cc @rynowak, since you reopened https://github.com/aspnet/Routing/issues/435

Reply:
We should do this for now.